SU-SWS/stanford_profile
# D8CORE-3973: added shared tags vocabulary.

# READY

# Summary
- This PR adds a new vocabulary, "Shared Tags".  It adds the appropriate fields to Basic Pages, and the various structured content types.  It hides the content in the shared tags field from the displays for all content types.  There are some acceptance tests as well.

# Needed By (Date)
- End of the sprint.

# Urgency
- This is functionality we've promised for 2.2.0

# Steps to Test

1. checkout branch
2. drush cim
3. verify the shared tags taxonomy exists.
4. verify the shared tags are added to the forms for basic pages and all the structured content types.
5. verify the shared tags do not appear on the user-facing pages of the site.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- D8CORE-3973, D8CORE-3974, D8CORE-3975

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
